src/main/java/org/openstreetmap/atlas/geography/atlas/complete/EmptyAtlas.java

```diff
@@ -193,23 +192,7 @@ public Edge edge(final long identifier)
     @Override
     public AtlasEntity entity(final long identifier, final ItemType type)
     {
-        switch (type)
-        {
-            case NODE:
-                return new CompleteNode(identifier);
-            case EDGE:
-                return new CompleteEdge(identifier);
-            case AREA:
-                return new CompleteArea(identifier);
-            case LINE:
-                return new CompleteLine(identifier);
-            case POINT:
-                return new CompletePoint(identifier);
-            case RELATION:
-                return new CompleteRelation(identifier);
-            default:
-                throw new CoreException("Unknown ItemType {}", type);
-        }
+        throw new UnsupportedOperationException();
```

Comment:
Whyyy?

Reply:
We could put it back, but then we need to arbitrarily locate the returned `CompleteEntities`. Maybe null island? This is part of that idea that we should never allow a `Located` to return `null` from a `bounds` call. I don't think it is reasonable for clients and users of `Located` objects to be forced to `null` check the result of `bounds` at every turn. I am open to having my mind changed on this. But as it stands, enforcing this will make the FeatureChange merging code easier to write I think.